src/Microsoft.PowerShell.Commands.Utility/commands/utility/Import-LocalizedData.cs

```diff
@@ -167,6 +167,7 @@ protected override void ProcessRecord()
                 Context.LanguageMode = PSLanguageMode.RestrictedLanguage;
                 try
                 {
+                    scriptBlock.ScriptBlockData.ForceMaliciousCodeScan = true;
                     result = scriptBlock.InvokeReturnAsIs();
```

Comment:
Assuming you don't squash this PR, this change seems to be in the wrong commit.

Reply:
Why? I check for `ForceMaliciousCodeScan` before doing AMSI/suspicious code scan when the script block is from a `.psd1` file. The reason to have this new property in `CompiledScriptBlockData` is to avoid scanning `.psd1` file unless it's being evaluated from `Import-LocalizedData`.

Reply:
Hmm, CodeFlow might be lying to me, it shows as a diff b/w iterations 3 and 4, but it also shows the diff in iteration 1.